apps/ios/Shared/Views/Database/DatabaseEncryptionView.swift

```diff
@@ -58,7 +58,7 @@ struct DatabaseEncryptionView: View {
     private func databaseEncryptionView() -> some View {
         List {
             Section {
-                settingsRow("key") {
+                settingsRow(storedKey ? "key.fill" : "key", color: storedKey ? .green : .secondary) {
```

Comment:
There's a minor bug on removing passphrase from keychain - toggle and icon initially become "turned off", but after going back to Database View w/t closing the sheet, the "Database passphrase" icon is still key, and when re-entering the view "Save passphrase in keychain" icon is green and toggle is on.